### PR TITLE
Added in the use cases UC01 - UC04

### PR DESCRIPTION
**Pull request by TheodoreKooo**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -306,16 +306,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-**Use case: Delete a client**
+**Use case: UC01 - Add a client's contact in SWEE**
 
 **MSS**
 
-1.  User requests to list clients
-2.  AddressBook shows a list of clients
-3.  User requests to delete a specific client in the list
-4.  AddressBook deletes the client
+1.  User inputs details of the client.
```
Comment (GiselleTan) [inline docs/DeveloperGuide.md:315]: should there be another step before this? such as "User requests to add a client's contact"?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -306,16 +306,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+2.  SWEE adds the client as a contact including the details given.
```
Comment (GiselleTan) [inline docs/DeveloperGuide.md:316]: perhaps we can remove "including the details given"

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -306,16 +306,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `SWEE` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a client**
+**Use case: UC01 - Add a client's contact in SWEE**
```
Comment (GiselleTan) [inline docs/DeveloperGuide.md:311]: could we leave out "in SWEE"?

**Review by GiselleTan — COMMENTED** [review]
Other than the comments, the rest LGTM!

**Review by camille-readbean — APPROVED** [review]
@GiselleTan 
Create a new issue for this and we add to v1.2 backlog instead as improvements.

Im gonna merge it as it is right now since this is technically okay for v1.1 #7.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (TheodoreKooo) [reply docs/DeveloperGuide.md:315]: ok will add that in, thanks

**Pull request merged**